page.codeberg.libre_menu_editor.LibreMenuEditor.yml

```diff
@@ -0,0 +1,42 @@
+app-id: page.codeberg.libre_menu_editor.LibreMenuEditor
+runtime: org.gnome.Platform
+runtime-version: "44"
+sdk: org.gnome.Sdk
+command: libre-menu-editor
+finish-args:
+  - --env=FLATPAK_APP_DIR=/app/libre-menu-editor
+    # to know when app is running as flatpak
+    # to know where files are stored
+  - --share=ipc
+    # to support x11-fallback
+  - --device=dri
+    # to avoid problems with graphics
+  - --socket=wayland
+    # to provide wayland support
+  - --socket=fallback-x11
+    # to support x11 if wayland is not available
+  - --filesystem=xdg-data/applications
+    # for reading and writing .desktop files in .local/share/applications
+  - --filesystem=xdg-data/flatpak:ro
+    # for reading .desktop files in .local/share/flatpak/exports/share/applications
+    # to avoid unreachable symlink targets in .local/share/flatpak
+  - --filesystem=/var/lib/flatpak:ro
+    # for reading .desktop files in /var/lib/flatpak/exports/share/applications
+    # to avoid unreachable symlink targets in /var/lib/flatpak
+  - --filesystem=/var/lib/snapd:ro
+    # for reading .desktop files in /var/lib/snapd/desktop/applications
+    # to avoid unreachable symlink targets in /var/lib/snapd
+  - --filesystem=host:ro
+    # for reading .desktop files in /usr/local/share/applications
+    # for reading .desktop files in /usr/share/applications
+    # to avoid unreachable symlink targets
+modules:
+  - name: git
+    buildsystem: simple
+    build-commands:
+      - cd libre-menu-editor; for file in $(find .); do install -D $file /app/libre-menu-editor/$file; done
+      - cd export; for file in $(find .); do install -D $file /app/$file; done
```

Comment:
I am of the opinion that this should be upstream, part of the `make install`. (all it needs it to honour prefix)

Reply:
I am sorry, I don't understand. I don't use "make install" anywhere

Reply:
and that's what I am saying. The install procedure should be in the upstream repository.

It's not blocking but it make the maintenance harder. 

(also it's very cryptic)

Reply:
If there is no problem, I would like to leave it this way